apps/ledger-live-mobile/src/components/RootNavigator/AnalyticsOptInPromptNavigator.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useMemo } from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { useTheme } from "styled-components/native";
+import { useFeature } from "@ledgerhq/live-config/featureFlags/index";
+import { LlmAnalyticsOptInPromptVariants } from "@ledgerhq/types-live";
+import { ScreenName } from "~/const";
+import { getStackNavigatorConfig } from "~/navigation/navigatorConfig";
+import AnalyticsOptInPromptMainA from "~/screens/AnalyticsOptInPrompt/variantA/Main";
+import AnalyticsOptInPromptDetailsA from "~/screens/AnalyticsOptInPrompt/variantA/Details";
+import AnalyticsOptInPromptMainB from "~/screens/AnalyticsOptInPrompt/variantB/Main";
+import AnalyticsOptInPromptDetailsB from "~/screens/AnalyticsOptInPrompt/variantB/Details";
+import { AnalyticsOptInPromptNavigatorParamList } from "./types/AnalyticsOptInPromptNavigator";
+
+export default function AnalyticsOptInPromptNavigator() {
+  const { colors } = useTheme();
+  const stackNavigationConfig = useMemo(() => getStackNavigatorConfig(colors, true), [colors]);
+  const llmAnalyticsOptInPromptFeature = useFeature("llmAnalyticsOptInPrompt");
+
+  const screensByVariant = {
+    [LlmAnalyticsOptInPromptVariants.variantA]: {
+      main: AnalyticsOptInPromptMainA,
+      details: AnalyticsOptInPromptDetailsA,
+    },
+    [LlmAnalyticsOptInPromptVariants.variantB]: {
+      main: AnalyticsOptInPromptMainB,
+      details: AnalyticsOptInPromptDetailsB,
+    },
+  };
+
+  const activeVariant =
+    llmAnalyticsOptInPromptFeature?.params?.variant === LlmAnalyticsOptInPromptVariants.variantB
+      ? LlmAnalyticsOptInPromptVariants.variantB
+      : LlmAnalyticsOptInPromptVariants.variantA;
+
+  return (
+    <Stack.Navigator screenOptions={stackNavigationConfig}>
+      <Stack.Screen
+        name={ScreenName.AnalyticsOptInPromptMain}
+        component={screensByVariant[activeVariant].main}
+        options={{ title: "", headerRight: undefined }}
+      />
+      <Stack.Screen
+        name={ScreenName.AnalyticsOptInPromptDetails}
+        component={screensByVariant[activeVariant].details}
+        options={{ title: "", headerRight: undefined }}
```

Comment:
Why do we need to set `headerRight` to `undefined`? What is the default value?

Reply:
By default the headerRight is a Close button component so we need to set it to undefined in our case

Reply:
Fixed (second param of getStackNavigatorConfig to false prevent Close buttons renderings, maybe we should plan to transform the props of getStackNavigatorConfig to an object so it's clearer when using it)